metals/src/main/scala/scala/meta/internal/metals/WorkspaceSearchVisitor.scala

```diff
@@ -86,10 +86,14 @@ class WorkspaceSearchVisitor(
   ): Option[SymbolDefinition] = {
     val nme = Classfile.name(filename)
     val tpe = Symbol(Symbols.Global(pkg, Descriptor.Type(nme)))
-    index.definition(tpe).orElse {
+
+    val forTpe = index.definitions(tpe)
```

Comment:
Does `forTpe` actually mean something here that I'm unaware of? Would it be more clear to just call this `symbolDefenitions` or something?

Reply:
It means that it tries to find a definition for typeSymbol.
If it's empty it also tries to find termSymbol.
Unfortunately, there is no equivalent to `orElse` combinator for empty List so I had to extract it into val